src/main/src/main/java/com/shevtsov/domain/Client.java

```diff
@@ -74,11 +72,16 @@ public void setPhone(String phone) {
     public String toString() {
         return "Client{" +
                 "id=" + id +
-                ", name='" + name + '\'' +
                 ", surname='" + surname + '\'' +
+                ", name='" + name + '\'' +
                 ", age=" + age +
-                ", email='" + email + '\'' +
                 ", phone='" + phone + '\'' +
+                ", email='" + email + '\'' +
                 '}';
     }
-}
+
+    @Override
+    public int compareTo(Client o) {
```

Comment:
where you used this?

Reply:
in ClientServiceImpl.getAll(). after Fryday`s testing I`v tryed to use Comparable and Comparator in our Project